Makefile

```diff
@@ -27,7 +27,9 @@ lint:
 	golangci-lint run -v
 
 .PHONY: build
-build: create-build-dir lint unit-test
+build: create-build-dir build-minimal lint unit-test
```

Comment:
build: create-build-dir lint unit-test build-minimal
to keep the build command for last as it was before...